docs/BreakingChanges/060 System.Web.UI.WebControls.EntityDataSource control binding invocations.md

```diff
@@ -0,0 +1,31 @@
+## 60: System.Web.UI.WebControls.EntityDataSource control
```

Comment:
{
  "Id": "60",
  "Title": "System.Web.UI.WebControls.EntityDataSource control",
  "Details": "The `Page.LoadComplete` event no longer causes the System.Web.UI.WebControls.EntityDataSource control to invoke data binding for changes to create/update/delete parameters.\nThis change eliminates an extraneous trip to the database, prevents the values of controls from being reset, and produces behavior that is consistent with other data controls, such as SqlDataSource and ObjectDataSource.\nThis change produces different behavior in the unlikely event that applications rely on invoking data binding in the `Page.LoadComplete` event.",
  "Suggestion": "TODO",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#asp",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis - API NOT IN DATABASE"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": "This change produces different behavior in the unlikely event that applications rely on invoking data binding in the Page.LoadComplete event.",
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] Not detectable via API analysis - API NOT IN DATABASE